### DISCUSSION TITLE
Is possible to print expression in languages other than english? (set a different locale)

### DISCUSSION THREAD

**Discussion by kompre** [sympy/sympy#24341] [Q&A]
Hello,

I'm a new user to sympy and I am poking around to see how it works and can use to produce technical documentations.

I've stumbled upon the `Piecewise()` function and I like it very much, but I the rendered output (latex, IPython) uses English terms (e.g. "for", "otherwise"). Since I need to write documentation in Italian it would be very nice If I could set up a locale different from the English default. I've searched a bit for such option, but didn't found any. I think this should be handled a the printing level?

For example I use another package, `Pint`, that handles different locale settings leveraging `babel` so that it can displays units name in any language. I was wondering if something similar is done also for sympy.

**Comment by moorepants** [comment]
You can always create your own printer. Here is one example: https://docs.sympy.org/latest/modules/printing.html#printer-method-example
